src/main/java/frc/robot/subsystems/claw/ClosedCone.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.claw;
+
+import frc.robot.RobotMap;
+import frc.robot.lib.statemachine.State;
+import frc.robot.lib.statemachine.Transition;
+
+/** Add your docs here. */
+public class ClosedCone extends State {
+    
+    @Override
+    public void build() {
+        // open claw if open request
+        transitions.add(new Transition(() -> {
+            return RobotMap.openRequest;
+        }, ClawStateMachine.openState));
+    }
+
+    @Override
+    public void init() {
+
+    }
+
+    @Override
+    public void execute() {
+        RobotMap.clawLib.setClawClosedCone();
```

Comment:
We do not need to execute this multiple times. The command to set the motor to a position only needs to occurr once.